tests/components/covid19/DataSourcesAndMethodologiesPage-test.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * DataSourcesAndMethodologiesPage-test.jsx
+ * Created by Lizzie Salita 2/11/21
+ */
+
+import React from 'react';
+import DataSourcesAndMethodologiesPage from 'components/covid19/DataSourcesAndMethodologiesPage';
+import { render, screen } from '@test-utils';
+
+const nativeDate = Date.now;
+
+afterAll(() => {
+    // restore the native date function
+    Date.now = nativeDate;
+});
+
+describe('COVID-19 Data Sources and Methodologies page', () => {
```

Comment:
instead of doing these nested describes, we can just do a single it block. When you execute the test it will print out the file it's testing to the console so we don't have to do this IMO.

Reply:
That's a good point and I can simplify the descriptions going forward. These tests can be deleted after 3/3 so I'll leave them as-is here!